src/components/transactions/TxDetails/TxData/DecodedData/index.tsx

```diff
@@ -50,7 +50,7 @@ export const DecodedData = ({ txData, toInfo }: Props): ReactElement | null => {
   const avatar = addressInfo?.logoUri || toInfo?.logoUri || txData.to.logoUri
 
   let decodedData = <></>
-  if (txData.dataDecoded) {
+  if (txData.dataDecoded && (method || (!method && txData.dataDecoded.parameters))) {
```

Comment:
this check for the parameters and method name is needed because for a native token transfer (from transaction builder) the decoded data comes also with an empty method: 

```js
{
    "type": "GENERIC",
    "method": "",
    "parameters": [
        {
            "name": "to",
            "type": "address",
            "value": "0xDa5e9FA404881Ff36DDa97b41Da402dF6430EE6b"
        },
        {
            "name": "value",
            "type": "uint256",
            "value": "1000000000000000000"
        }
    ]
}
```